Makefile

```diff
@@ -93,7 +93,7 @@ golangci-lint/%: | $(GOLANGCI_LINT)
 
 .PHONY: build
 build: generate
-	$(GOCMD) build -o otel-go-instrumentation ./cli/...
+	CGO_ENABLED=0 $(GOCMD) build -o otel-go-instrumentation ./cli/...
```

Comment:
Can we change this to use a conditionally set var? E.g.

```make
CGO_ENABLED?=0

.PHONY: echo
echo:
	CGO_ENABLED=$(CGO_ENABLED) go env CGO_ENABLED
```

That way it can still be set from the command line if wanted, just it would have a different default:

```terminal
$ make echo
CGO_ENABLED=0 go env CGO_ENABLED
0
$ CGO_ENABLED=1 make echo
CGO_ENABLED=1 go env CGO_ENABLED
1
```

Reply:
Updated, PTAL